apple/OmnivoreKit/Sources/Utils/ImageCache.swift

```diff
@@ -27,24 +27,29 @@ public final class ImageCache {
   }
 
   private let queue = DispatchQueue(label: "app.omnivore.image.cache.queue", attributes: .concurrent)
-  private let cache = NSCache<AnyObject, PlatformImage>()
+  private let cache = NSCache<NSString, PlatformImage>()
 
   private init() {
-    cache.totalCostLimit = 1024 * 1024 * 50 // 50 MB
+    // cache.totalCostLimit = 1024 * 1024 * 1024 * 50 // 50 MB
+    print("CREATING IMAGE CACHE: ", self)
   }
 
   private func image(_ url: URL) -> PlatformImage? {
     var cachedImage: PlatformImage?
     queue.sync {
-      cachedImage = cache.object(forKey: url as AnyObject)
+      cachedImage = cache.object(forKey: NSString(string: url.absoluteString))
```

Comment:
@satindar I'll remove these debugging lines, but I discovered when we use `URL` as the key the cache is never hit. I guess its not doing a proper equality check or something.

Reply:
Oh wow! Guess I never tested this thoroughly 🙃

Reply:
I wonder if it used to? Also it might not even matter because the NSURLCache stuff could be used when we try to re-download. 

Kind of weird though, I didn't see anything on that blog post about it.